src/v/archival/ntp_archiver_service.h

```diff
@@ -76,7 +77,8 @@ class ntp_archiver {
       ss::lw_shared_ptr<const configuration> conf,
       cloud_storage::remote& remote,
       cloud_storage::cache& c,
-      cluster::partition& parent);
+      cluster::partition& parent,
+      ss::shared_ptr<cloud_storage::async_manifest_view> amv);
```

Comment:
This commit probably doesn't build since the construction site of the archival is updated in the next commit.